src/main/java/picard/fingerprint/ExtractFingerprint.java

```diff
@@ -97,7 +97,7 @@ protected int doWork() {
         }
 
         checker.setLocusMaxReads(LOCUS_MAX_READS);
-        checker.setReferenceFasta(REFERENCE_SEQUENCE);
+        checker.setReferenceFasta(referenceSequence.getReferenceFile());
```

Comment:
Isn't parseArgs doing this?https://github.com/broadinstitute/picard/blob/4c937e5bb8b790a5b7096c1e61564633e83298f0/src/main/java/picard/cmdline/CommandLineProgram.java#L359
(or are you removing deprecated usage?)

Reply:
I got a report that without this fix the tool was unable to open cram files. I didn't verify this claim though...so I should probably look at the tests....